geometry_msgs/Twist move
geometry_msgs/Pose marker_pose
---
bool finished
---
float32 distance
